customize

```diff
@@ -31,17 +31,21 @@ set_apt_sources() {
     cat <<EOF >> ${ROOTDIR}/etc/apt/sources.list
 # Repository: $SUITE
 deb $APT_MIRROR $SUITE $COMPONENTS
-deb-src $APT_MIRROR $SUITE $COMPONENTS
```

Comment:
The deb-src is a repository that include the source packages needed to build or compile packages. I don't think that this will be need in lepidopter.